app/components/search/SearchResults/SearchResults.module.scss

```diff
@@ -179,3 +157,60 @@
     display: block;
   }
 }
+
+@media print {
+  // This below styles are for printing out the results on our service/search results page.
+
+  // This sets visibility hidden on all elements except for the searchResultsContainer
+  // and its children to avoid printing irrelevant sections of the page.
+  // This follows the example of SO this answer https://stackoverflow.com/a/2618980
+  body {
```

Comment:
See comment in description about using this solution to show only the search results in the print view.